### PR TITLE
upgrade CSI to 1.1.0

### PR DESCRIPTION
**Pull request by jichenjc**
update CSI to 1.1.0

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by jichenjc** [comment]
https://github.com/container-storage-interface/spec/releases

File: pkg/csi/cinder/controllerserver.go [hunk 1]
```diff
@@ -394,6 +394,10 @@ func (cs *controllerServer) GetCapacity(ctx context.Context, req *csi.GetCapacit
 	return nil, status.Error(codes.Unimplemented, "")
 }
 
+func (cs *controllerServer) ControllerExpandVolume(ctx context.Context, req *csi.ControllerExpandVolumeRequest) (*csi.ControllerExpandVolumeResponse, error) {
+	return nil, status.Error(codes.Unimplemented, "")
```
Comment (ramineni) [inline pkg/csi/cinder/controllerserver.go]: could we add msg here as well, "ControllerExpandVolume is not yet implemented"

File: pkg/csi/cinder/controllerserver.go [hunk 2]
```diff
@@ -387,11 +387,15 @@ func (cs *controllerServer) ControllerGetCapabilities(ctx context.Context, req *
 }
 
 func (cs *controllerServer) ValidateVolumeCapabilities(ctx context.Context, req *csi.ValidateVolumeCapabilitiesRequest) (*csi.ValidateVolumeCapabilitiesResponse, error) {
```
Comment (ramineni) [inline pkg/csi/cinder/controllerserver.go]: import fmt is missing

**Comment by adisky** [comment]
@jichenjc please update patch and address @ramineni's comment, we want to have it soon

**Comment by jichenjc** [comment]
ok, thanks for the reminder @adisky

**Comment by ramineni** [comment]
format job still needs to be fixed , @jichenjc could you run hack/update-gofmt.sh , i think imports order needs to be changed

**Comment by jichenjc** [comment]
@adisky my home network  don't allow me to connect to slack channel today, I saw the message you sent in slack (by mail notification) but can't reply to you .... sorry, so I updated the PR just now

**Comment by adisky** [comment]
> @adisky my home network don't allow me to connect to slack channel today, I saw the message you sent in slack (by mail notification) but can't reply to you .... sorry, so I updated the PR just now and will keep enhancing to make pass test

@jichenjc no problem, thanks for the update

**Pull request merged**